src/core/qgsvectorlayer.cpp

```diff
@@ -2192,6 +2192,17 @@ bool QgsVectorLayer::readSymbology( const QDomNode &layerNode, QString &errorMes
       }
     }
 
+    mIncludeAttributesOapifTemporalFilters.clear();
```

Comment:
Shouldn't this be in QgsVectorLayerServerProperties ?

Reply:
no, this is the same leftover of the other include variable, to explain, I first implemented it as an explicit whitelist filter for OAPIF but then I decided to reuse the same temporal settings already in place for WMS-T because they already have begin/end support and that is exactly the same logic we need here, so I just added `Date` options to handle date filters (not datetime). I'll remove the whole code.